core/admin/start.py

```diff
@@ -64,7 +64,7 @@ def test_DNS():
 
 cmdline = [
     "gunicorn",
-    "--threads", f"{os.cpu_count()}",
+    "--threads", f"{os.environ.get('CPU_COUNT', 1)}",
```

Comment:
No need for a f-string:
```suggestion
    "--threads", os.environ.get('CPU_COUNT', '1'),
```